.github/workflows/opcode-logger-testing.yml

```diff
@@ -0,0 +1,50 @@
+name: Opcode logger testing
+
+on:
+  pull_request:
+    branches: [ main, release/** ]
+  push:
+    branches: [ main, release/** ]
+    tags: [ v* ]
+
+jobs:
+  check:
+    name:
+      Opcode logger comparison between besu and hedera
+    runs-on: [self-hosted, Linux, large, ephemeral]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Use Node.js [18.15]
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 18.15
+          cache: npm
+
+      - name: Create .env file
+        run: cp local.env .env
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Upgrade @hashgraph/hedera-local to v2.27.1
+        run: npm install @hashgraph/hedera-local@2.27.1 --save
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@8f1998e9878d786675189ef566a2e4bf24869773 # v1.2.0
+        with:
+          version: nightly
+
+      - name: Run besu node
```

Comment:
Q: why do we need to run besu again.
Is it that the `@OpcodeLogger` tests rely on comparing outputs from besu and local node?

Reply:
Gotcha. I'd lie this called out in some doc somewhere so people understand the flow